src/book/02-system/01-foundations/06-config-script.mdx

```diff
@@ -40,16 +43,19 @@ However, it makes no sense for the default configuration file to contain the
 
 </Alert>
 
+The platform-specific override is intended to allow for different configuration values between real robots and webots. For instance, the real NUgus robot uses a different Visual Mesh network compared to the robots run in the Webots simulator. 
+
 The robot-specific override is intended to allow configuration values to be tweaked to account a slight variation in the robots hardware. For instance, the cameras on each robot have different serial numbers.
 
 The role-specific override is intended to change a modules behaviour based on which role is running. For instance, disabling the transmission of certain NUsight2 traffic while playing a competition game.
 
 To reiterate how the configuration system loads and merges configuration files
 
 1. The default configuration file `config/ModuleName.yaml` is loaded. This forms the base configuration for the module
-1. A check is made to see if `config/RobotName/ModuleName.yaml` exists. If it does, it is loaded and all values found therein override the equivalent values in the base configuration
-1. A check is made to see if `config/RoleName/ModuleName.yaml` exists. If it does, it is loaded and all values found therein override the equivalent values in the base configuration
-1. If any of these 3 configuration files are modified then 1-3 are run again
+2. A check is made to see if `config/PlatformName/ModuleName.yaml` exists. If it does, it is loaded and all values found therein override the equivalent values in the base configuration
+3. A check is made to see if `config/RobotName/ModuleName.yaml` exists. If it does, it is loaded and all values found therein override the equivalent values in the base configuration
+4. A check is made to see if `config/RoleName/ModuleName.yaml` exists. If it does, it is loaded and all values found therein override the equivalent values in the base configuration
```

Comment:
```suggestion
2. A check is made to see if `config/<PlatformName>/<ModuleName>.yaml` exists. If it does, it is loaded and all values found therein override the equivalent values in the base configuration
3. A check is made to see if `config/<RobotName>/<ModuleName>.yaml` exists. If it does, it is loaded and all values found therein override the equivalent values in the base configuration
4. A check is made to see if `config/<RoleName>/<ModuleName>.yaml` exists. If it does, it is loaded and all values found therein override the equivalent values in the base configuration
```

Reply:
I feel like this might be too messy with the number of things, plus we'd have to do it below for scripts. I feel like this is more of an example, following the example above.